android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -1182,6 +1215,34 @@ class HmssdkFlutterPlugin :
         }
     }
 
+    private val keyChangeListener = object : HMSKeyChangeListener {
```

Comment:
It's possible that apps can attach multiple `HMSKeyChangeListener` for different or even same **keys**. 
With this implementation, it cannot be supported.